src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,87 @@
+.calendar {
+
+  &__container {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+
+    height: 100vh;
+  }
+
+  &__title {
+    display: flex;
+    justify-content: center;
+
+    @extend %calendar-font;
+    color: black;
+    cursor: pointer;
+
+    &:hover {
+      text-shadow: 3px 3px pink;
+    }
+  }
+
+  &__second-title {
+    display: flex;
+    justify-content: center;
+
+    @extend %calendar-font;
+    font-size: 15px;
+  }
+
+  &__content {
+    display: flex;
+    flex-wrap: wrap;
+    gap: $day-gap;
+
+    width: $calendar-width;
+
+    padding: 35px $calendar-padding;
+
+    margin: 0 auto;
+  }
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-grow: 0;
+    flex-shrink: 0;
+    flex-basis: $day-square-size;
+
+    height: $day-square-size;
+
+    text-align: center;
+    border: 1px solid black;
+
+    background-color: $day-color;
+
+    cursor: pointer;
+    transition: 0.5s;
+
+    &:hover {
+      box-shadow: 5px 5px gray;
+      background-color: pink;
+      transform: translate(-5px, -20px);
+    }
+
+    @for $index from 1 through 31 {
+      &:nth-child(#{$index})::before {
+        content: "#{$index}";
+        @extend %calendar-font;
```

Comment:
Don't use SASS loops for styles that stay the same for all elements of the group or for the most part of page.